app/docs/async-api.yaml

```diff
@@ -0,0 +1,80 @@
+asyncapi: 2.4.0
+info:
```

Comment:
So I am basing my comments off this object being what this channel receives from `ffc-pay-return`:

{
   "currency":"GBP",
   "detail":"",
   "frn":1000000001,
   "invoiceNumber":"S0000001SFIP000001V001",
   "reference":"PY1234567",
   "settled":true,
   "settlementDate":"2021-08-27T00:00:00.000Z",
   "sourceSystem":"SITIAgri",
   "value":50000
}